examples/example_multilabel_classification.py

```diff
@@ -0,0 +1,75 @@
+"""
+=================================
+example_multilabel_classification
+=================================
+
+This examples shows how to format the targets for a multilabel classification
+problem. Details on multilabel classification can be found on
+`here https://scikit-learn.org/stable/modules/multiclass.html>`_).
+"""
+import numpy as np
+
+import sklearn.model_selection
+import sklearn.datasets
+import sklearn.metrics
+from sklearn.utils.multiclass import type_of_target
+
+
+import autosklearn.classification
+
+
+############################################################################
+# Data Loading
+# ============
+
+# Using reuters multilabel dataset -- https://www.openml.org/d/40597
+X, y = sklearn.datasets.fetch_openml(data_id=40594, return_X_y=True, as_frame=False)
+
+# fetch openml downloads a numpy array with TRUE/FALSE strings. Re-map it to
+# integer dtype with ones and zeros
+y[y == 'TRUE'] = 1
```

Comment:
Shouldn't Auto-sklearn encode the data itself?

Reply:
We can make this happen. Today is not possible because input validation happens for automl.

Nevertheless, the estimators perform a check-targets early in the game that throws:
ValueError: classification with data of type multiclass-multioutput is not supported

I will do a push to showcase the required changes to support this.